3rd-party/woocommerce-services.php

```diff
@@ -73,6 +73,9 @@ public function error_notice() {
 	 * @return bool result of installation/activation
 	 */
 	private function install() {
+		$jetpack = Jetpack::init();
+		$jetpack->stat( 'jitm', 'wooservices-activated-' . JETPACK__VERSION );
```

Comment:
I'd rather track an additional stat to determine successful install.. as it sits, we should be able to glean that people are having installation problems by comparing the number of `activated` stats to active sites.. though that method isn't without it's flaws.

What do you think about tracking an `install` stat on button click, and an `activated` stat on successful result of the `install()` method?